feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableHeader.kt

```diff
@@ -31,9 +32,14 @@ fun TimetableHeader(modifier: Modifier = Modifier) {
                 style = MaterialTheme.typography.labelMedium,
             )
         }
+        val droidKunInBathDrawableRes = if (isSystemInDarkTheme()) {
```

Comment:
Oops, I just learned that this part is unnecessary and will fix it immediately.
Modify to use drawable-night.
https://developer.android.com/guide/topics/resources/providing-resources?hl=ja

Reply:
Thanks for your kindly comment. you have provided us with good insight 😉